src/components/Search/SearchView.jsx

```diff
@@ -4,13 +4,31 @@ import Box from '@mui/material/Box';
 import Container from '@mui/material/Container';
 import Typography from '@mui/material/Typography';
 
+import zap from '../../theme/img/zap-10.svg';
 import SearchInput from './SearchInput';
 
 export default function SearchView() {
   return (
     <Container maxWidth="lg">
       <Box>
-        <Typography variant="h1" component="div" align="center" gutterBottom>
+        <Typography
+          variant="h1"
+          component="div"
+          align="center"
+          gutterBottom
+          sx={{
+            '&:after': {
+              backgroundImage: `url(${zap})`,
+              backgroundPosition: '55%',
+              backgroundRepeat: 'no-repeat',
+              backgroundSize: '25%',
+              content: '""',
+              display: 'block',
+              height: '0.3em',
+              marginTop: '-5px',
+            },
+          }}
+        >
```

Comment:
rather than try to use the protocol npm package, I am hard-coding this for now as it was the easier route, and I don't imagine we'll be needing a lot of what's included.